### ISSUE TITLE
Address UHD blu-ray ripping/encoding in documentation

### ISSUE THREAD

**Issue by zcutlip** [lisamelton/video_transcoding#338]
I'm not sure if anything special needs to be done for UHD blu-rays either on the ripping side, or when using `transcode-video`. Particularly as it applies to fancy stuff like HDR, dolby vision, & atmos

I've ripped a couple UHD blu-rays and encoded them using the same options I've always used, and they *seem* to be fine. But I wouldn't be surprised to learn at some point there was a better way, or I missed something

Anyway, a mention of this in the documentation would be useful, even if only to say nothing special is required compared to "normal" blu-ray rips.

Thanks!

**Comment by donmelton** [comment]
@zcutlip Good idea. This project is currently no longer under active development. I have a new project here which is much better suited for transcoding 4K content and HDR:

https://github.com/donmelton/other_video_transcoding

An upcoming version of the `other-transcode` tool from that project will actually copy HDR10 metadata when transcoding.

**Comment by zcutlip** [comment]
Okay, that's good to know, thank you.

I haven't tried `other-transcode` yet since I do all my transcoding on a Mac mini. This caveat wiki suggests maybe macOS is less than ideal:

> [Apple VideoToolbox](https://developer.apple.com/documentation/videotoolbox) is the only hardware video encoder available on macOS. And although it can produce good quality, it's not possible to further improve that quality via tuning.

**Comment by donmelton** [comment]
@zcutlip Which macOS is less than ideal in terms of tweaking, the quality, especially on the newer Apple Silicon-based models, is quite good. Plus, `other-transcode` can use the `x264` and `x265` software encoders as well. If you want to use `x264`, I strongly recommend the `--x264-cbr` option.
